lib/casino/ldap_authenticator.rb

```diff
@@ -77,17 +77,16 @@ def user_filter(username)
   end
 
   def extra_attributes(user_plain)
-    if @options[:extra_attributes]
```

Comment:
Is it safe to remove this conditional?

Alternatively, you could do `(@options[:extra_attributes] || []).each_with_object({})...`